ansible_base/authentication/serializers/authenticator_map.py

```diff
@@ -27,14 +24,30 @@ def validate(self, data) -> dict:
         if map_type == 'organization' and (not org or org == ''):
             errors["organization"] = "You must specify an organization with the selected map type"
 
-        if not data.get('order', None):
-            errors['order'] = "Must be a valid integer"
-
         if errors:
             raise ValidationError(errors)
         return data
 
-    def validate_trigger_data(self, triggers: dict, definition, error_prefix: str) -> dict:
+    def validate_trigger_data(self, data):
+        errors = {}
+        request = self.context.get('request', None)
+        if not request or (request.method != 'PATCH'):
```

Comment:
It's difficult to image that `_validate_trigger_data` should be called differently when changed in PATCH vs PUT.

Reply:
Like, if it is a PATCH and "triggers" is in data, then you should call `_validate_trigger_data`.

Reply:
switched conditions, now it should be ok